### ISSUE TITLE
Last message cut-off at bottom of archived day of messages

### ISSUE THREAD

**Issue by grahamperrin** [gitterHQ/gitter#1508]
Screenshots at https://discourse.trueos.org/t/pros-and-cons-of-gitter/110/3?u=grahamperrin

> … I key 'Page down' until I can go no further. Partial or complete disappearance of the scroller. Critically: the foot of the page is invisible. … 

> … Pagination problems and scrolling problems on the Gitter side may partly explain the indexing failures on the Google side. …

Would you like a separate issue for disappearance of the scroller?

**Comment by grahamperrin** [comment]
Compare the screenshot at https://github.com/gitterHQ/gitter/issues/1507#issuecomment-260118199 (apparently no chat after **19:47** yesterday) with this shot of an archive view of yesterday, where it appears that nothing was written after **18:41** –

![2016-11-11 apparently ending at 18-41](https://cloud.githubusercontent.com/assets/192271/20237629/3a9b5c26-a8d0-11e6-828e-b123b6c19aba.png)

Wonder whether problems with pagination are contributory to the symptoms that are observed in #1507.

**Comment by grahamperrin** [comment]
> … 19:47 yesterday … 18:41 

Chromium views of things. I launched Chromium, visited the URL: 

![2016-11-12 12-18 chromium](https://cloud.githubusercontent.com/assets/192271/20237802/2d14c8a4-a8d4-11e6-8cac-f4ded2bce250.png)

– that's **20:47 (Chromium) a.k.a. 19:47 (Firefox)**; for the purposes of this issue I'll ignore the discrepancies between representations of time. 

At the foot of the window there's the date and time stamp but the message content is invisible; the required scroll bar is not present. 

After making the window a little less shallow, for a scroller to appear, I scroll down as far as possible and there's apparently nothing after **19:41**: 

![2016-11-12 12-24 chromium](https://cloud.githubusercontent.com/assets/192271/20237823/ffc7af3c-a8d4-11e6-8081-04f304f7ddc2.png)

– and from discussion in IRC (not bridged) with someone who's in that _Lobby_, I'm certain that this Chromium view of things is wrong. After making the window much more shallow and again, scrolling down as far as possible: 

![2016-11-12 12-28 chromium](https://cloud.githubusercontent.com/assets/192271/20237821/f91d31ca-a8d4-11e6-9242-fd4db937764a.png)

**Comment by grahamperrin** [comment]
Safari 10.0.1 (12602.2.7) with a pre-release build of Sierra: 

![screen shot 2016-11-12 at 12 50 25](https://cloud.githubusercontent.com/assets/192271/20237888/b97761a6-a8d6-11e6-8999-69ec88d088f7.png)

![screen shot 2016-11-12 at 12 51 04](https://cloud.githubusercontent.com/assets/192271/20237891/c33ff7f2-a8d6-11e6-9fb0-b31c25de3ecc.png)

```
sh-3.2$ date ; sw_vers
Sat 12 Nov 2016 12:49:27 GMT
ProductName:	Mac OS X
ProductVersion:	10.12.1
BuildVersion:	16B2327e
sh-3.2$ 
```

----

## In all cases

Wherever there's a scroller to the right of, and adjacent to, the messages: 

- the foot of the scroller becomes invisible.

**Comment by MadLittleMods** [comment]
For future issues, etc, please boil down your issue into a concise, clear, description alongside some detailed reproduction steps. It's hard to follow multiple posts trying to explain the issue.

This could easily be explained by, when viewing an archived day of messages with a screen short enough to overflow, there are some messages cut off at the bottom.

**Comment by grahamperrin** [comment]
Sorry – when I opened the issue I knew that there was a problem but I wasn't sure about steps to make it reproducible. 

----

Incidentally, whilst typing this comment in Firefox I see that the scroller at the edge of the page can similarly lose its foot when I scroll to the foot of the page.
